packages/fxa-content-server/app/scripts/templates/confirm_signup_code.mustache

```diff
@@ -15,8 +15,8 @@
         <p class="verification-email-message">{{#unsafeTranslate}}Please enter the verification code that was sent to %(escapedEmail)s within 5 minutes.{{/unsafeTranslate}}</p>
         <form novalidate>
             <div class="input-row">
-                <!-- Using `type="text" inputmode="numeric"` shows the numericpad on mobile and strips out whitespace on desktop. -->
-                <input type="text" inputmode="numeric" pattern="\d[ ]*" class="tooltip-below otp-code" placeholder="{{#t}}Enter 6-digit code{{/t}}" required autofocus />
+                <!-- Not using `type="text" inputmode="numeric"` because of https://bugzilla.mozilla.org/show_bug.cgi?id=1205133 -->
+                <input type="number" inputmode="numeric" pattern="\d[ ]*" class="tooltip-below otp-code" placeholder="{{#t}}Enter 6-digit code{{/t}}" required autofocus />
```

Comment:
With this change empty /whitespace padded codes come up like this

<img width="442" alt="Screen Shot 2020-02-28 at 4 20 22 PM" src="https://user-images.githubusercontent.com/1295288/75588388-90192900-5a46-11ea-978d-2ab77a3c1ba2.png">

With original version it would appear as a tooltip on input field.

Reply:
@ryanfeeley Do we need to support stripping out the whitespace for the input field? Removing that requirement might make this bug a little easier to fix?

Reply:
Yeah, the pattern attribute isn't supported on number inputs. 😞   I'm fine with WONTFIX this also, since the real problem is https://bugzilla.mozilla.org/show_bug.cgi?id=1205133